ambry-messageformat/src/main/java/com.github.ambry.messageformat/BlobPropertiesSerDe.java

```diff
@@ -27,21 +27,33 @@ public SystemMetadata(long blobSize, String serviceId, String ownerId, String co
     super(blobSize, serviceId, ownerId, contentType, isPrivate, timeToLiveInSeconds);
     this.creationTimeInMs = creationTime;
   }
+
+  public SystemMetadata(long blobSize, String serviceId, String ownerId, String contentType, boolean isPrivate,
```

Comment:
yes, those changes need to go into the patch where in we enable write in new format